dao/mongo.go

```diff
@@ -195,7 +207,7 @@ func (m *MongoService) CreateAppointmentResource(appointmentResourceDao *models.
 }
 
 // UpdateInsolvencyPractitioners updates the practitoners for an Insolvency Case
-func (m *MongoService) UpdateInsolvencyPractitioners(practitionersResource models.InsolvencyResourceDto, transactionID string) (int, error) {
+func (m *MongoService) UpdateInsolvencyPractitioners(practitionersResource models.InsolvencyResourceDao, transactionID string) (int, error) {
```

Comment:
The variable practitionersResource would be better named insolvencyResource or similar

Reply:
Fixed.Thank you